eogrow/pipelines/sampling.py

```diff
@@ -154,7 +154,7 @@ def _get_output_features(self) -> List[FeatureSpec]:
         return output_features
 
 
-class BaseRandomSamplingPipeline(BaseSamplingPipeline, metaclass=abc.ABCMeta):
+class BaseRandomSamplingPipeline(BaseSamplingPipeline, metaclass=abc.ABCMeta):  # noqa B024
```

Comment:
This one is above my pay-grade

Reply:
this has to do with ABCs and inheritance. If you're inheriting from an abstract class, but wish to remain abstract (i.e. not implement the methods, just add some functionalities), flake8 doesn't realize that your class still has some abstract methods (inherited as abstract and never implemented) and marks it as a false abstract class